src/sonic-config-engine/data/l3intf.j2

```diff
@@ -0,0 +1,65 @@
+{
+    "DEVICE_METADATA": {
+        "localhost" : {
+        "hwsku" : "{{ DEVICE_METADATA.localhost.hwsku }}"
+    }
+    },
+    {% set ns = {'firstPrinted': False} -%}
+    "PORT": {
+        {%- for key, value in PORT.items() -%}
+        {%- if ns.firstPrinted %},{% endif %}
+
+        "{{ key }}": {
+            {%- for keyPort, valuePort in value.items() %}
+
+            {% if keyPort != "admin_status" %}"{{ keyPort }}": "{{ valuePort }}",{% endif %}
+            {%- endfor %}
+
+            "admin_status": "{{ value.admin_status|default("up") }}",
+            "mtu": "9100"
+        }
+        {%- if ns.update({'firstPrinted': True}) %}{% endif -%}
+        {%- endfor %}
+
+    },
+    "LOOPBACK_INTERFACE": {
+        "Loopback0": {},
+        "Loopback0|10.1.0.1/32": {}
+    },
+    "BGP_NEIGHBOR": {},
+    "DEVICE_NEIGHBOR": {},
+    "INTERFACE": {
+        "Ethernet0": {},
+        "Ethernet1": {},
+        "Ethernet2": {},
+        "Ethernet3": {},
+        "Ethernet4": {},
+        "Ethernet5": {},
+        "Ethernet6": {},
+        "Ethernet7": {},
+        "Ethernet8": {},
```

Comment:
Since this is an l3 template did you want to add IP addresses to these interfaces in your usecase?

Reply:
> @VenkatCisco, can you remove this file **src/sonic-config-engine/.config_samples.py.swp** from this PR ?

Sure, removed.

Reply:
> Since this is an l3 template did you want to add IP addresses to these interfaces in your usecase?

In the review mtg, there was an ask from Guhon and team to not hardcode the ip address etc. Even though the config_samples.py has other areas like generate_t1_sample_config catered to specific needs, this method is made more generic as per request.